Stripe/UI/STPPaymentCardTextField.m

```diff
@@ -403,6 +403,55 @@ - (STPCardParams *)card {
     return c;
 }
 
+- (void)setCard:(nonnull STPCardParams *)card {
```

Comment:
:+1: added tests in https://github.com/stripe/stripe-ios/commit/003d702e001e99b8ccac0b0a7c501c81b8e97a9a